src/main/java/com/trainignapp/trainingapp/model/Trainer.java

```diff
@@ -1,39 +1,13 @@
 package com.trainignapp.trainingapp.model;
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.Getter;
```

Comment:
These two imports are not used, you can delete them. 
When comitting with IDEA you can check the box "Optimize imports and it will make that for you"